metashare/api/jobs.py

```diff
@@ -192,6 +205,17 @@ def create_branches_on_github_then_create_scratch_org(
 )
 
 
+def check_if_changes_on_org(*, scratch_org, user):
+    with report_errors_on_check_changes(scratch_org):
+        scratch_org.has_changes = sf_org_changes.sf_org_has_changes(
+            scratch_org=scratch_org, user=user
+        )
```

Comment:
Maybe you're still intending to do this, but I think we discussed replacing the boolean `has_changes` with the actual nested `changes` object (or `null` if no changes).

https://trello.com/c/8uR5ZTuC/36-i-want-to-capture-changes-for-a-task-at-any-point-in-the-process#comment-5d8100b20ec7da142ac3360a

Reply:
Yep, getting there bit by bit; I can't capture _what the changes are_ yet.